thrift_streaming/exitserver/go.mod

```diff
@@ -0,0 +1,52 @@
+module exitserver
+
+go 1.20
+
+replace github.com/cloudwego/kitex-tests => ../..
```

Comment:
can remove?

Reply:
No. It's compiled when generating but the `go.mod` of this project is not available at that time so it has an individual `go.mod`